doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -467,58 +467,309 @@ The following table indicates the software maturity levels of the support for ea
         - nRF9151
         - nRF9160
         - nRF9161
-      * - **Bluetooth LE Peripheral/Central**
+      * - **2 Mbps PHY**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Supported
         - Experimental
         - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Coded PHY (Long Range)**
+        - --
+        - --
+        - --
+        - Supported
         - Supported
         - Supported
         - Supported
         - Experimental
+        - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Concurrent Roles**\ :sup:`1`
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Experimental
+        - Supported
+        - --
         - --
         - --
         - --
+      * - **Data Length Extensions**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Experimental
+        - Supported
+        - --
         - --
-      * - **Connectionless/Connected CTE Transmitter**
         - --
+        - --
+      * - **Advertising Extensions**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Experimental
+        - Supported
+        - --
         - --
+        - --
+        - --
+      * - **Periodic Advertising with Responses**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Experimental
         - Supported
         - --
+        - --
+        - --
+        - --
+      * - **Periodic Advertising Sync Transfer**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Experimental
+        - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Isochronous Channels**
+        - Supported\ :sup:`2`
+        - Supported\ :sup:`2`
+        - Supported
+        - Supported\ :sup:`2`
+        - Supported
+        - Supported\ :sup:`2`
+        - Supported
         - Experimental
+        - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Direction Finding**\ :sup:`3`
```

Comment:
for the 52 series, the following devices support DFE:
- 52811
- 52820
- 52833